src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  let sum = days * pricePerDay;
+
+  if (days >= 7) {
+    sum -= 50;
+  } else if (days >= 3) {
+    sum -= 20;
+  }
```

Comment:
You should return value right away after you've found the right condition. It makes the code simpler to read and helps to avoid unnecessary `else if`
```suggestion
  if (days >= 7) {
    return sum - 50;
  } 
  
  if (days >= 3) {
    return sum - 20;
  }
```